react/__mocks__/productMock.js

```diff
@@ -0,0 +1,37 @@
+export const productMock = [
+  {
```

Comment:
I'd prefer to save this as a json file. what do you think?

Reply:
makes sense